From dac74aaf557d3c14e4d934bc5cdb7c13e816e493 Mon Sep 17 00:00:00 2001
From: Wenlong Zhang <zhangwenlong@loongson.cn>
Date: Wed, 29 Jun 2022 16:20:08 +0800
Subject: [PATCH] add loong64 support for panicwrap

Signed-off-by: Wenlong Zhang <zhangwenlong@loongson.cn>
---
 dup2.go | 1 +
 dup3.go | 2 +-
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/dup2.go b/dup2.go
index d9c61c7..2772c53 100644
--- a/dup2.go
+++ b/dup2.go
@@ -1,4 +1,5 @@
 // +build linux,!arm64 !linux,!windows
+// +build linux,!loong64
 
 package panicwrap
 
diff --git a/dup3.go b/dup3.go
index 9417bc7..9632f08 100644
--- a/dup3.go
+++ b/dup3.go
@@ -1,4 +1,4 @@
-//+build linux,arm64
+//+build linux,arm64 linux,loong64
 
 package panicwrap
 
